libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -151,6 +153,14 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
 
     @Input()
     compact = false;
+    /**
+     *attribute [cozy]="true" to hide class fd-form-layout-grid-container that reset padding 
+     */
+    @Input()
+    cozy: boolean;
```

Comment:
Since the component has `compact` input, `cozy` looks redundant. Let's try to follow up on the steps from the comments below

Reply:
1. Get rid of it